components/sidebar.js

```diff
@@ -99,30 +99,18 @@ const LoadingImage = styled.img`
 export default ({ hackathons, currentPath }) => {
   const [loading, setLoading] = useState(false);
   const [ifTimeOut, setIfTimeOut] = useState();
-  const [trailingPath, setTrailingPath] = useState();
-
-  const getTrailingPath = () => {
-    const paths = window.location.href.split('/');
-    const pathIndex = paths.findIndex((val) => val === currentPath);
-    const trailingPathArray = paths.slice(pathIndex + 1);
-    if (pathIndex === -1 || trailingPathArray.length === 0) {
-      return 'intro';
-    }
-    return trailingPathArray.join('/');
-  };
 
   useEffect(() => {
     setLoading(false);
     clearTimeout(ifTimeOut);
-    setTrailingPath(getTrailingPath);
   }, [window.location.pathname]);
 
   const generateLinkTemplate = () => {
-    return `/[id]/${trailingPath}`;
+    return `/[id]/intro`;
```

Comment:
why did this change to intro?

Reply:
shouldn't `getTrailingPath()` return `/intro` when there is no trailing path anyways?

Reply:
problem is that `Livesite/announcements` returns `announcements` as trailing path, and there is no `[id]/announcements`

Reply:
wouldn't this remove the feature of navigating to a different hackathon but staying on the same section?

Reply:
why not just check if the currentpath is Livesite and return a intro or something in the getTrailingPath function?

Reply:
oh word didn't consider that. Also why is that the functionality? Seems weird to me

Reply:
Its mostly a feature for the sponsorship team. If they're switching hackathons, it usually means they need to add sponsors to multiple hackathons, so it removes a few clicks for them. For our logs teams, they probably will never switch hackathons, so its shouldn't effect them.